x/auction/keeper/auctions_test.go

```diff
@@ -0,0 +1,245 @@
+package keeper_test
```

Comment:
I'd like to see test coverage increase here. Main cases for each auction:
Same bidder, valid bid.
New bidder, valid bid
Add tests for main invalid bid cases